index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please follow the design guidelines and work in the opacity for your program section to make your mobile version similar to the original version :smile:
#### Original design
![DDDDDD](https://user-images.githubusercontent.com/6615394/181406140-64fbacad-4f03-4152-b9ad-c7862fc83de1.png)

### Your design
![Screenshot from 2022-07-28 02-15-17](https://user-images.githubusercontent.com/6615394/181406145-89775512-2890-487b-ac07-d2537fba9801.png)
![Screenshot from 2022-07-28 02-21-39](https://user-images.githubusercontent.com/6615394/181406428-7563a9eb-9f49-4724-a61e-d03ce3f4c0ad.png)